Directory.Build.props

```diff
@@ -1,19 +1,32 @@
 <Project ToolsVersion="15.0">
 
   <PropertyGroup>
-    <VersionPrefix>0.0.4</VersionPrefix>
     <Authors>@jet @amjjd @eiriktsarpalis and contributors</Authors>
     <Company>Jet.com</Company>
     <Description>Simple version-tolerant Newtonsoft.Json Converters for F# types</Description>
     <PackageProjectUrl>https://github.com/jet/Jet.JsonNet.Converters</PackageProjectUrl>
     <PackageTags>fsharp unionconverter JsonPickler json converters</PackageTags>
     <PackageLicenseUrl>https://github.com/jet/Jet.JsonNet.Converters/blob/master/LICENSE</PackageLicenseUrl>
-    <Copyright>Copyright © 2016-8</Copyright>
-
+    <Copyright>Copyright © 2016-8</Copyright>    
+    <RepositoryType>git</RepositoryType>
```

Comment:
sourcelink should be adding these, no ?

Reply:
ah, I see now, can you move this down to the mono override section ? use a <Choose to bracket things